### DISCUSSION TITLE
关于新的播放器置顶插件

### DISCUSSION THREAD

**Discussion by gingerbreap** [the1812/Bilibili-Evolved#4297] [功能建议 / Ideas]
想实现如同旧版置顶插件那样可以将up面板往下放置于弹幕池和推荐视频之间，但是自己试了一下宽屏模式下弹幕池会藏在视频后方……不知道有没有更好的解决办法……

另外，现在的版本标题移动至按钮上方时中间有一个很怪的留白，在我的设备上设置`style="height:80px;"`效果会显著改善，尝试缩放也没有问题，不知道能不能作为改善建议……？

看到这个用习惯了的插件终于复活了真的很感动（

**Comment by the1812** [comment]
帮你 at 下作者 @rien7
